tom_desc/settings.py

```diff
@@ -316,6 +324,7 @@
     'tom_alerts.brokers.scout.ScoutBroker',
     'tom_alerts.brokers.tns.TNSBroker',
     'tom_fink.fink.FinkBroker',
+    'tom_pittgoogle.pittgoogle.PittGoogleBroker',
```

Comment:
I can import our TOM `Broker` here, but I'm not actually using it in what I'm proposing in this PR.

Reply:
in the case of FInk, the broker is what creates a GenericAlert object out of the stream it receives.

Reply:
then the stream command like fink.py just translates it into a Target object

Reply:
I am not sure what the goal is of the parsers dir is, but I am still learning the arcane of how TOM/Django put all these different pieces in music.

Reply:
I just looked at your branch. It looks to me like the parser and the broker are just two different ways to accomplish the same goal. Your way of using the TOM broker seems cleaner to me, but I'm happy to do whatever you all prefer.

Reply:
Yes, so there are clearly many ways to deal with it, perhaps too many :) We need to define an architecture, and we are in the middle of experimenting actually. So whatever seems faster/easier will be fine for now. But indeed ideally the <broker>_client would eventually shield away from tom_desc any specifics of that broker in managing the alerts and provide a unified agnostic API to the tom_desc. Of course the problem is that the tom_desc developers, completely newbies, are far away from the expertise needed to be able to define the proper API :)